dojo/tools/aqua/parser.py

```diff
@@ -19,9 +19,19 @@ def get_findings(self, json_output, test):
 
     def get_items(self, tree, test):
         items = {}
-        if "resources" in tree:
+        if isinstance(tree, list):
+            vulnerabilityTree = tree[0]["results"]["resources"]
```

Comment:
Instead of duplicating all the code related to walking the tree, why not just set the `vulnerabilityTree` variable based on the format we received? A brief comment explaining why this is necessary also wouldn't hurt. Finally, there should be a length check on `tree` to make sure the reference to `[0]` won't cause an index out-of-bounds error on an empty report.